### DISCUSSION TITLE
How can I enable for the users to create their own custom filters?

### DISCUSSION THREAD

**Discussion by davinhun** [YetiForceCompany/YetiForceCRM#16881] [How to] [answered]
Hello

As an adminnistrator I can create my own filter in every module, I have the following create filter button:
![image](https://user-images.githubusercontent.com/20862325/218306614-52f2f3aa-9f87-41f4-b72b-49762d882b0d.png)

But my users don't have it
![image](https://user-images.githubusercontent.com/20862325/218306635-16f3e80a-c8f8-4aa7-a23a-bbac4d82c70b.png)

Where can I enable them for every module to create ther own filter?

**Comment by apcloic** [comment, marked as answer]
Hello @hunterk43
Check this :
https://github.com/YetiForceCompany/YetiForceCRM/issues/3460#issuecomment-243509964
